src/components/UpgradeStatus.tsx

```diff
@@ -1,52 +1,12 @@
 import React from "react"
-import styled from "@emotion/styled"
 import { TranslationKey } from "../utils/translations"
 import Translation from "./Translation"
+import { Heading, Text, VStack } from "@chakra-ui/react"
 
 export interface IStyledContainer {
   isShipped: boolean
 }
 
-const Container = styled.div<IStyledContainer>`
-  background: ${(props) =>
-    props.isShipped
-      ? props.theme.colors.upgradeStatusShippedBackground
-      : props.theme.colors.upgradeStatusBackground};
-  border: ${(props) =>
-    props.isShipped
-      ? props.theme.colors.upgradeStatusShippedBorder
-      : props.theme.colors.upgradeStatusBorder};
```

Comment:
This is a weird case where we are not going to be able to just do `borderColor=upgradeStatusBorder` because those variables are not colors 🤷🏼 

Take a look at them in `src/theme.ts` (the old theme):
```
// we were doing
upgradeStatusBorder: `${primaryLight200} solid 2px`,
upgradeStatusShippedBorder: `${success400} solid 2px`,
```

What we need to do is something like 
```tsx
border="2px" borderColor={success400}
```

Not sure if Im clear. LMK if you need help on this.